src/test/java/org/fundacionjala/at15/katas/banckorc/validAccountTest.java

```diff
@@ -0,0 +1,65 @@
+package org.fundacionjala.at15.katas.banckorc;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.fundacionjala.at15.katas.bankocr.ValidAccount;
+import org.junit.Test;
+
+public class validAccountTest {
+
+    @Test
+    public void itShouldBeValidAccount() {
+        boolean valid = true;
+        ValidAccount vAccount = new ValidAccount("D:\\workspace\\coding\\AT15\\src\\main\\java\\org\\fundacionjala\\at15\\katas\\bankocr\\txtFile\\validAccount.txt");
+        assertEquals(valid, vAccount.getValidation());
+    }
+
+    @Test
+    public void shouldBeAccountsToValidate() {
+        List<String> exp = new ArrayList<>();
+        //added for the test
+        exp.add("345882865");
+        exp.add("000000051");
+        //changed for the test
+        exp.add("49006771x");
+        //changed for the test
+        exp.add("12345678x");
+        ValidAccount vAccount = new ValidAccount("D:\\workspace\\coding\\AT15\\src\\main\\java\\org\\fundacionjala\\at15\\katas\\bankocr\\txtFile\\validAccount.txt");
+        assertEquals(exp, vAccount.getAccountsToValidate());
```

Comment:
Even when your tests pass, the validation is checking "hardcoded" logic in your ValidAccount class. The path sent to the constructor of ValidAccount seems to be irrelevant